packages/core/src/Image/Image.stories.args.ts

```diff
@@ -1,58 +0,0 @@
-import { colors, borderRadii, rounded, shadows } from '../storybook/args'
-export const argTypes = {
```

Comment:
We've been working to move args out of story files and into these separate arg files. We should add to this file rather than delete it.